README.md

```diff
@@ -0,0 +1,12 @@
+# NodeJS mentoring program
+
+## Prerequisites
+* node - ^12.18.3
+* yarn - ^1.21.1
```

Comment:
During executions of yarn install or start  program using yarn (v.1.22.4) I got the following error:
```
PS D:\Mentoring_NodeJS\Module_1\Code-Review\nodejs-mentoring> yarn start:1.1
yarn run v1.22.4
error node-mentoring@1.0.0: The engine "yarn" is incompatible with this module. Expected version "1.21". Got "1.22.4"
error Commands cannot run with an incompatible environment.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```
But I see that it can be yarn v.1.21.1 or higher

Reply:
updated engines property in package.json file